modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/constants.js

```diff
@@ -12,14 +12,41 @@
  * details.
  */
 
+/**
+ *	Cart implementation constants
```

Comment:
We might autoformat this in the future, but everywhere else in DXP (and even elsewhere in this PR) we only have one space between the `*` and the comment contents.